src/frontend/posts_page/components/CommentInputBar.tsx

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import {connect} from "react-redux";
+import {addComment, addLike, undoLike} from "../actions";
+import {IPost} from "./PostBlock";
+
+interface ICommentInputBarProps{
+    addComment: any,
+    post: IPost
+}
+interface ICommentInputBarState{
+    comment: string,
+    checked: boolean,
```

Comment:
what is `checked` ?

Reply:
Check if set private checkbox is checked or not